### PR TITLE
Add polyfill for Number.parseFloat and Number.parseInt.

### LINKED ISSUE #980: Number.parseInt/parseFloat not supported in older browsers.
Number.parseInt/parseFloat have the same functionality as the global parseInt/parseFloat and are part of ECMAScript 2015. Number.parseInt/float are not supported in IE and Safari based on the info from MDN:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseInt

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseFloat

Would be great if the polyfill have something like:
if (!Number.parseInt) {
Number.parseInt = window.parseInt
}

Same for parseFloat:
if (!Number.parseFloat) {
Number.parseFloat = window.parseFloat;
}

### PR DESCRIPTION
**Pull request by ruiaraujo**
Fixes #980.

### CODE REVIEW

File: polyfills/Number/parseInt/config.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+	"aliases": [
+		"es6"
+	],
+	"browsers": {
+		"ie": "<12",
+		"ie_mob" : "*",
+		"firefox": "<34",
```
Comment (JakeChampion) [inline polyfills/Number/parseInt/config.json]: MDN says it was in version 25 of firefox

Comment (ruiaraujo) [reply polyfills/Number/parseInt/config.json]: I mixed Chrome and firefox. Both were wrong. Fixed now. tested in browserstack to be super sure.

**Review by JakeChampion — DISMISSED** [review]

**Comment by ruiaraujo** [comment]
I just noticed it is missing the samsung browser in the config.json, now the question is:
polyfill.io only support SB 4 which does not need this polyfill since it is forked of Chromium 44 but older version would need it. Should I add it as documentation?

**Comment by JakeChampion** [comment]
@ruiaraujo There are failing tests on CI for these changes:

```
Failures:
 • ie/11 (targeted) [https://saucelabs.com/tests/7bd28a27475449d7b1a0a72d752d3588]
    -> Number.parseInt
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=Number.parseInt
    -> Number.parseFloat
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=Number.parseFloat
 • ie/10 (targeted) [https://saucelabs.com/tests/5791072f17ef49e581e62ea30302efba]
    -> Number.parseInt
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=Number.parseInt
    -> Number.parseFloat
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=Number.parseFloat
 • ie/9 (targeted) [https://saucelabs.com/tests/98df88c9e3424364929790f175dd8d7b]
    -> Number.parseInt
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=Number.parseInt
    -> IntersectionObserver
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=IntersectionObserver
    -> Number.parseFloat
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=Number.parseFloat
 • ie/8 (targeted) [https://saucelabs.com/tests/1c5af72962cf46e5a2b6808905831384]
    -> Number.parseInt
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=Number.parseInt
    -> Number.parseFloat
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=Number.parseFloat
 • ie/7 (targeted) [https://saucelabs.com/tests/61c769b0508c478dad993d96b5e427f6]
    -> Number.parseInt
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=Number.parseInt
    -> Number.parseFloat
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=Number.parseFloat
 • safari/8 (targeted) [https://saucelabs.com/tests/bc72e943fb4246b28f95ca3a1d2aa9b0]
    -> Number.parseInt
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=Number.parseInt
    -> Number.parseFloat
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=Number.parseFloat
 • android/4.4 (targeted) [https://saucelabs.com/tests/d8a7713deb434526b6f3e3877ecb3af4]
    -> Number.parseInt
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=Number.parseInt
    -> Number.parseFloat
       http://127.0.0.1:3000/test/tests?mode=targeted&feature=Number.parseFloat
```

**Review by JakeChampion — APPROVED** [review]

**Pull request merged**